common/src/main/java/com/mx/common/configuration/package-info.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Path Core - Configuration
+ *
+ * <p>Path Core provides a rich configuration layer to streamline the definition and discovery of configuration bindings
+ * along with definable validations
+ *
+ * <p>Configuration binding draws values in the gateway.yml "configurations" nodes into a configuration POJO that can then
```

Comment:
Do we care that you've written gateway.yml here instead of gateway.yaml? I know we support either, just wondering if we have a preferred version.